src/mongocxx/collection.cpp

```diff
@@ -47,18 +47,6 @@
 #include <mongocxx/result/update.hpp>
 #include <mongocxx/write_concern.hpp>
 
-namespace {
-enum class cursor_flag : uint32_t {
```

Comment:
Probably the latter... we have a partial duplicate of this in `src/mongocxx/options/find.hpp`